test/gtest/ucp/test_ucp_rma.cc

```diff
@@ -63,16 +72,17 @@ class test_ucp_rma : public test_ucp_memheap {
     }
 
 protected:
-    void test_mem_types(send_func_t send_func) {
+    void test_mem_types(send_func_t send_func, bool proto_supported) {
         std::vector<std::vector<ucs_memory_type_t> > pairs =
                 ucs::supported_mem_type_pairs();
 
         for (size_t i = 0; i < pairs.size(); ++i) {
 
             // TODO remove this check after memory types is fully supported by
             // RMA API
-            if (!UCP_MEM_IS_ACCESSIBLE_FROM_CPU(pairs[i][0]) ||
-                !UCP_MEM_IS_ACCESSIBLE_FROM_CPU(pairs[i][1])) {
+            if (!(enable_proto() && proto_supported) &&
```

Comment:
did you mean `if (!(enable_proto() || !proto_supported) &&` ?

Reply:
no, added comment to explain